### DISCUSSION TITLE
Bidirectional pagination starting from a given date

### DISCUSSION THREAD

**Discussion by ippocratis** [Radiokot/photoprism-android-client#54] [Ideas]
https://github.com/Radiokot/photoprism-android-client/issues/53

**Comment by leosamuele221** [comment]
https://github.com/bleibdirtroy/MobilePrism This app do scrolling in the "right" way like Google Photo. Maybe the code can be helpfull

**Reply by Radiokot** [reply]
Thanks, Samuele. This app downloads everything to the local storage, it is not the way I want to go with my gallery.
